src/main/java/com/demo/service/PlansService.java

```diff
@@ -14,6 +16,10 @@
     private final PlansDao plansDao;
 
     public List<Plans> getPlans(String planType) {
```

Comment:
Can we use an enum for planType and use switch cases here? is that a possibility?

Reply:
Enum created for planType. But, switch cases compared with String for enum. So, used enums and iteration for validation. Suggest of any changes.